Modules/Devices/src/Devices.Infrastructure/Persistence/Repository/IdentityRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Entities;
+using Backbone.Modules.Devices.Infrastructure.Persistence.Database;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.Persistence.Database;
+using Enmeshed.BuildingBlocks.Application.Extensions;
+using Enmeshed.BuildingBlocks.Application.Pagination;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backbone.Modules.Devices.Infrastructure.Persistence.Repository;
+public class IdentityRepository : IIdentityRepository
+{
+    private readonly DbSet<Identity> _identityDbSet;
+    private readonly IQueryable<Identity> _readonlyIdentityDbSet;
```

Comment:
I usually like expressive naming, but in this case, I think we can keep it simple:

```suggestion
    private readonly DbSet<Identity> _identities;
    private readonly IQueryable<Identity> _readonlyIdentities;
```